File: RWTech/PointLine BioProx - C/RWTech/RWTech/BioproxC/RWTech/Common/EquipamentoManager.cs
using BioproxC.RWTech.Pojo;

namespace BioproxC.RWTech.Common
{
    public class EquipamentoManager
    {
        private static EquipamentoManager instance;
        private Equipamento equipamento;

        public static EquipamentoManager getInstance()
        {
            if (instance == null)
            {
                instance = new EquipamentoManager();
            }
            return instance;
        }

        public Equipamento getEquipamento()
        {
            if (equipamento == null)
            {
                equipamento = new Equipamento();
            }
            return equipamento;
        }

        public void setEquipamento(Equipamento relogio)
        {
            this.equipamento = relogio;
        }
    }
}

File: RWTech/PointLine BioProx - C/Aplicação_Exemplo_Java/Código Fonte/SimuladorIntegracaoREP/src/br/com/rwtech/simulador/common/EquipamentoManager.java
package br.com.rwtech.simulador.common;

import br.com.rwtech.simulador.pojo.Equipamento;

public class EquipamentoManager {
	
	private static EquipamentoManager instance;
	private Equipamento equipamento;
	
	public static EquipamentoManager getInstance() {
		if (instance == null) {
			instance = new EquipamentoManager();
		}
		return instance;
	}

	public Equipamento getEquipamento() {
		if (equipamento == null) {
			equipamento = new Equipamento();
		}
		return equipamento;
	}
	
	public void setEquipamento(Equipamento relogio) {
		this.equipamento = relogio;
	}
}
